cpu/opcodes_d0.c

```diff
@@ -64,11 +64,37 @@ static int aiv_gb_opcode_d5(aiv_gameboy *gb)
     return 16;
 }
 
+//SUB d8
+static int aiv_gb_opcode_d6(aiv_gameboy *gb)
+{
+    u8_t n1 = gb->a;
+    u8_t n2 = aiv_gb_memory_read8(gb, gb->pc);
+    gb->pc++;
+
+    gb->a = n1 - n2;
+
+    aiv_gb_set_flag(gb, ZERO, gb->a == 0);
+    aiv_gb_set_flag(gb, NEG, 1);
+    aiv_gb_set_flag(gb, HALF, (((n1 & 0x0f) - n2) & 0xf0));
+    aiv_gb_set_flag(gb, CARRY, n1 >= n2);
+
```

Comment:
Se ti interessa in opcodes_90 c 'è void _aiv_gb_sub_internal(aiv_gameboy *gb, u8_t to_sub) che si occupa di effettuare la sottrazione e settare tutte e 4 le flags, la utilizzo già per tutte le altre sub e sbc ed è testata